src/js/helpers.js

```diff
@@ -58,3 +58,8 @@ Handlebars.registerHelper('strip_label', function(label) {
     var result = label.substring(label.indexOf(':') + 1, label.length);
     return result.trim();
 });
+
+// Convert non alpha numeric characters to HTML ID friendly characters
+Handlebars.registerHelper('sanitize', function (obj) {
+    return obj.replace( /[^a-z0-9]+/gi, "_" );
```

Comment:
I think we can probably get away with escaping `+`, `#` and maybe `-`. I don't think we'll have any other character in our labels